package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "electron-localshortcut",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
I will bump minor, on master there is another change not yet published: the Accelerator validity check using `electron-is-accelerator`